File: Practice/src/introduction_coding_tests/MorseCode1.js
function solution(letter) {
    const morse = [
        '.-',
        '-...',
        '-.-.',
        '-..',
        '.',
        '..-.',
        '--.',
        '....',
        '..',
        '.---',
        '-.-',
        '.-..',
        '--',
        '-.',
        '---',
        '.--.',
        '--.-',
        '.-.',
        '...',
        '-',
        '..-',
        '...-',
        '.--',
        '-..-',
        '-.--',
        '--..',
    ];

    return letter
        .split(' ')
        .map((v, i) => String.fromCharCode(morse.indexOf(v) + 97))
        .join('');
}

File: Practice/src/introduction_coding_tests/MorseCode1.py
def solution(letter):
    morse = [".-", "-...", "-.-.", "-..", ".", "..-.", "--.", "....", "..", ".---", "-.-", ".-..", "--", "-.", "---",
             ".--.", "--.-", ".-.", "...", "-", "..-", "...-", ".--", "-..-", "-.--", "--.."]
    return "".join(chr(morse.index(i) + 97) for i in letter.split())
